src/function/table.rs

```diff
@@ -235,7 +220,10 @@ impl Table {
     /// removed.
     pub(crate) fn remove(&mut self, inp: &[Value], ts: u32) -> bool {
         let hash = hash_values(inp);
-        let Ok(entry) = self.table.find_entry(hash, search_for!(self, hash, inp)) else {
+        let Ok(entry) = self
+            .table
+            .find_entry(hash, search_for(&self.vals, hash, inp))
```

Comment:
this can just use `self.search_for`? (similarly above)

Reply:
It would cause a double borrow error